src/main/java/controller/MainController.java

```diff
@@ -148,18 +147,17 @@ private JTable createExcelTable() {
 	 * @author Lino Silva
 	 */
 	private void convertExcelRows() {
```

Comment:
Please document your fix in the javadoc ie explain why you started in 1 and not 0 :)